kettle-plugins/hdfs/src/main/java/org/pentaho/big/data/kettle/plugins/hdfs/trans/HadoopFileOutputMeta.java

```diff
@@ -105,9 +111,27 @@ protected String getProcessedUrl( IMetaStore metastore, String url ) {
     if ( url == null ) {
       return null;
     }
-    NamedCluster c = namedClusterService.getNamedClusterByName( sourceConfigurationName, metastore );
+    if ( metastore == null ) {
+      // Maybe we can get a metastore from spoon
+      if ( Spoon.getInstance() != null ) {
```

Comment:
@CodeOnCoffee Ok, changes are in to use the MetaStoreConst.  I didn't restart wingman on this because it will fail until https://github.com/pentaho/pentaho-kettle/pull/3682 is compiled.